.gitignore

```diff
@@ -3,3 +3,5 @@ node_modules
 test-results
 
 **/.DS_Store
```

Comment:
.DS_Store shouldn't be in the repo as per Professor's suggestions. Maybe you can keep it in your local repo.